### PR TITLE
Portuguese Language Updated

### PR DESCRIPTION
**Pull request by MatsuoYutani**
**When merged this pull request will:**
- Add about 600 lines translated to Portuguese
- Some lines were fixed and updated to make more sense

### CODE REVIEW

File: addons/advanced_ballistics/stringtable.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (commy2) [inline addons/advanced_ballistics/stringtable.xml]: Which one is right?

Comment (Blutze) [reply addons/advanced_ballistics/stringtable.xml]: @MatsuoYutani What tool did you use to make these edits, and if it was bux's Tabler, which version?

Comment (MatsuoYutani) [reply addons/advanced_ballistics/stringtable.xml]: Hello @commy2  and @Blutze I used Bux's Tabler latest version 0.9.1

Comment (commy2) [reply addons/advanced_ballistics/stringtable.xml]: @bux

Comment (bux) [reply addons/advanced_ballistics/stringtable.xml]: The new one has BOM, looks fine to me

Comment (dedmen) [reply addons/advanced_ballistics/stringtable.xml]: > The new one has BOM, looks fine to me.
> Indeed the ace one is missing the BOM

But we want without bom.

Comment (bux) [reply addons/advanced_ballistics/stringtable.xml]: Then the other way round :D
tabler checks wether the original file has BOM and only adds it in that case.
https://github.com/bux/tabler/commit/c67fd0dc69655878d30be6c0002f7ebdde928a82#diff-17e621989897af9f01c2714076d01920R154

Comment (MatsuoYutani) [reply addons/advanced_ballistics/stringtable.xml]: So... Do I have to remake it?

Comment (Blutze) [reply addons/advanced_ballistics/stringtable.xml]: On the "files changed" tab here on the PR, you can see in which files you caused changes to the very first line. If you go through them on your fork and open them the GitHub webclient's editor, you should be able to see a little red dot as the first character that you can remove like a normal one.

**Comment by commy2** [comment]
Thank you! :>

**Comment by commy2** [comment]
https://github.com/acemod/ACE3/issues/367
Portuguese is at 100% 🎉

**Pull request merged**